### DISCUSSION TITLE
Theme Config Help

### DISCUSSION THREAD

**Discussion by Tristan-Zaleski** [JanDeDobbeleer/oh-my-posh#2354] [Troubleshoot]
Have downloaded and been playing around with getting one of the custom themes working with my system:
Windows 10 computer
Hyper JS as my command line
bash shell

I have created my own theme.omp.json file, using the code provided for the atomic theme as my base. Downloaded a Nerd font and gotten icon's to appear by saving and setting the fontFamily but I have 2 issues: 
there appears to be dividing lines at the ends of each section of the prompt &
there is an unwanted smiley face plaintext icon where the cursor normally is, which will at times disappear on navigating to other folders. 
(Shown below)
![cmdIssue](https://user-images.githubusercontent.com/70818940/171092933-264e0029-4e0e-4fc7-aa88-c22170236e09.png)

(Code for end of prompt and arrows)
``` {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "foreground": "#21c7c7",
          "style": "plain",
          "template": "\u2570\u2500",
          "type": "text"
        },
        {
          "foreground": "#e0f8ff",
          "foreground_templates": ["{{ if gt .Code 0 }}#ef5350{{ end }}"],
          "properties": {
            "always_enabled": true
          },
          "style": "plain",
          "template": "\ue285\ufb00 ",
          "type": "exit"
        }
      ],
      "type": "prompt"
    }
  ],
  "osc99": true,
  "version": 2
}
```
(Code being used for first few segments)
```
"blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "background": "#0077c2",
          "foreground": "#ffffff",
          "leading_diamond": "\u256d\u2500\ue0b6",
          "style": "diamond",
          "template": "\uf120 {{ .Name }} ",
          "type": "shell"
        },
        {
          "background": "#ef5350",
          "foreground": "#FFFB38",
          "style": "diamond",
          "template": "<parentBackground>\ue0b0</> \uf292 ",
          "type": "root"
        },
        {
          "background": "#a7c2ab",
          "foreground": "#2d3436",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "folder_icon": " \uf07b ",
            "home_icon": "\uf7dd",
            "style": "full"
          },
          "style": "powerline",
          "template": " \uf07b\uf553{{ .Path }} ",
          "type": "path"
        },
```

Any way to remove the smiley face cursor thing and line breaks in the prompt, I would really appreciate some guidance on.

**Comment by JanDeDobbeleer** [comment]
@Tristan-Zaleski the lines are an [issue](https://github.com/vercel/hyper/issues/3575) in the terminal, not oh-my-posh. As for the smiley, I'm not seeing that when I use your config so maybe you have a setting in Hyper somewhere?

**Reply by Tristan-Zaleski** [reply]
Unfortunate to hear about the line issue, but not a deal breaker by any means. The smiley seems to show when I am in a directory under my designated user account on my machine. This was not something I experienced prior to using oh-my-posh and the Nerd Font, so I guess I will have to investigate that further. As you said, the line issues do not appear within a git bash terminal, nor the smiley face even in my user directories... Odd, but very much a Hyper related issue. Thank you so much for the speedy reply.
